llama_cpp/src/Utility/DataGenerator.h

```diff
@@ -45,10 +58,16 @@ class DataGenerator {
  public:
     DataGenerator() {}
 
-    template<typename T>
-    static void generateRandomData(std::vector<T> *targetVector, int count = DATA_GENERATOR_DEFAULT_COUNT);
+    template<typename T, typename Y>
+    static void generateRandomData(std::vector<T> *targetVector,
+                                   Y min = std::numeric_limits<Y>::min(),
+                                   Y max = std::numeric_limits<Y>::max(),
+                                   int count = DATA_GENERATOR_DEFAULT_COUNT);
 
-    static void generateRandomDataFromSet(const std::string& set, std::vector<std::string> *targetVector, int count);
+    static void generateRandomDataFromSet(std::vector<std::string> *targetVector,
```

Comment:
Why the function name is not talking about string but actually generate some strings?